httpreq/httpreq.go

```diff
@@ -12,26 +11,20 @@ import (
 	"github.com/pangpanglabs/goutils/behaviorlog"
 )
 
-const DefaultMaxIdleConnsPerHost = 100
-
-var defaultClient *http.Client
+type key int
 
-func init() {
-	defaultTransportPointer, ok := http.DefaultTransport.(*http.Transport)
-	if !ok {
-		panic(fmt.Sprintf("defaultRoundTripper not an *http.Transport"))
-	}
-	defaultTransport := *defaultTransportPointer
-
-	// http.DefaultMaxIdleConnsPerHost = 2
-	defaultTransport.MaxIdleConnsPerHost = DefaultMaxIdleConnsPerHost
+const (
+	dataTypeKey key = iota
```

Comment:
When use `dataTypeKey`? If we don't use it, please remove this `const` part.

Reply:
ok，I removed it